Tests/AppTests/__Snapshots__/AnalyzerTests/test_analyze.1.txt

```diff
@@ -1,97 +1,14 @@
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-1"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-1'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-1"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-1'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-1"
-  ▿ 'git fetch --tags' at path: '.../github.com-foo-2'
-    - command: "git fetch --tags"
-    - path: ".../github.com-foo-2"
-  ▿ 'git log --max-parents=0 -n1 --format=format:"%ct"' at path: '.../github.com-foo-2'
-    - command: "git log --max-parents=0 -n1 --format=format:\"%ct\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%H-%ct" "2.0.0"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%H-%ct\" \"2.0.0\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%H-%ct" "2.1.0"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%H-%ct\" \"2.1.0\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%H-%ct" "main"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%H-%ct\" \"main\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%ct"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%ct\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git reset "origin/main" --hard' at path: '.../github.com-foo-2'
-    - command: "git reset \"origin/main\" --hard"
-    - path: ".../github.com-foo-2"
-  ▿ 'git reset --hard' at path: '.../github.com-foo-2'
-    - command: "git reset --hard"
-    - path: ".../github.com-foo-2"
-  ▿ 'git rev-list --count HEAD' at path: '.../github.com-foo-2'
-    - command: "git rev-list --count HEAD"
-    - path: ".../github.com-foo-2"
-  ▿ 'git tag' at path: '.../github.com-foo-2'
-    - command: "git tag"
-    - path: ".../github.com-foo-2"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-2'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-2"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-2'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-2"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-2'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-2"
+▿ 13 elements
+  - "./github.com-foo-1: commitCount"
+  - "./github.com-foo-1: firstCommitDate"
+  - "./github.com-foo-1: lastCommitDate"
+  - "./github.com-foo-1: getTags"
+  - "./github.com-foo-1: revisionInfo for main"
+  - "./github.com-foo-1: revisionInfo for 1.0.0"
+  - "./github.com-foo-1: revisionInfo for 1.1.1"
+  - "./github.com-foo-1: checkout main"
+  - "./github.com-foo-1: dumpPackage"
+  - "./github.com-foo-1: checkout 1.0.0"
+  - "./github.com-foo-1: dumpPackage"
+  - "./github.com-foo-1: checkout 1.1.1"
+  - "./github.com-foo-1: dumpPackage"
```

Comment:
Mmmm, the whole point of snapshotting was that we'd capture the exact commands. I think the problem was that we were doing too much in that snapshot test: capturing both which commands ran in what order and what those commands look like.

Now we test what the commands look like in `ShellOutCommandExtensionTests`, which means this test can focus on testing the command sequence.

I was tempted to convert this away from snapshots but I think a snapshot is actually a better way to test this, because array diffs in the event of failure are going to be easier to read.